charts/connect/values.yaml

```
@@ -189,6 +191,12 @@ connect:
     # Number of profiler snapshots to keep.
     keepLast: 12
 
+  # 1Password Connect Custom Container Environment Variables
+  # Must be written in the following format:
+  # - name: VARIABLE_NAME
+  #   value: VARIABLE_VALUE
+  customEnvVars: {}
```

`[non-blocking]`

This definition seems to cause the following warning:
```
coalesce.go:286: warning: cannot overwrite table with non table for connect.connect.customEnvVars (map[])
```

A possible fix for this is to set this as an empty array:
```suggestion
  customEnvVars: []
```

---

Good catch! I didn't notice the error since I wasn't saving my output to a file, should be fixed now!